CollAction/CollAction.csproj

```diff
@@ -20,27 +21,7 @@ npm run build</PreBuildEvent>
   </PropertyGroup>
 
   <ItemGroup>
-    <Content Include="wwwroot\images\team\Brian Russell.png" />
```

Comment:
These where causing the build to break for me. I have already deleted them from my working copy.